### ISSUE TITLE
test_multi_select_double_click error

### ISSUE THREAD

**Issue by cdeil** [holoviz/panel#6902]
@hoxbro - I see this UI test error:

```
(base) panel $ pixi run test-ui
✨ Pixi task (_install-ui in test-ui): playwright install chromium

✨ Pixi task (test-ui in test-ui): pytest panel/tests/ui --ui --browser chromium -n logical --dist loadgroup --reruns 3 --reruns-delay 10
=================================================================================== test session starts ====================================================================================
platform darwin -- Python 3.12.3, pytest-7.4.4, pluggy-1.5.0
rootdir: /Users/cdeil/code/oss/panel
configfile: pyproject.toml
plugins: cov-5.0.0, github-actions-annotate-failures-0.2.0, playwright-0.5.0, rerunfailures-14.0, anyio-4.4.0, xdist-3.5.0, base-url-2.1.0, asyncio-0.23.6
asyncio: mode=Mode.AUTO
10 workers [529 items]  kipped                                                                                                                                                             
sssssssss.............s...........................................................................................................................................R......R.......... [ 33%]
......................R.........................................................................................................................R.x.R.xR.xx....R.................... [ 67%]
.......................x.....................R.xx..R.......x.....R.x.RR..........................xxx.xxRxxFxxxxxx..xxxxxx.xxxxx..........................................R.....RR..F [ 99%]
F..RF.                                                                                                                                                                               [100%]
========================================================================================= FAILURES =========================================================================================
______________________________________________________________________________ test_multi_select_double_click ______________________________________________________________________________
[gw5] darwin -- Python 3.12.3 /Users/cdeil/code/oss/panel/.pixi/envs/test-ui/bin/python3.12

page = <Page url='http://localhost:55237/'>

    def test_multi_select_double_click(page):
        clicks = []
        select = MultiSelect(options=['A', 'B', 'C'], on_double_click=clicks.append)
    
        serve_component(page, select)
    
        page.locator('option').nth(1).dblclick()
    
>       wait_until(lambda: clicks and clicks[0].option == 'B')
E       ValueError: `wait_until` callback must return None, True or False, returned []

panel/tests/ui/widgets/test_select.py:33: ValueError
```

Some others time out:
```
FAILED panel/tests/ui/widgets/test_select.py::test_multi_select_double_click - ValueError: `wait_until` callback must return None, True or False, returned []
FAILED panel/tests/ui/widgets/test_icon.py::test_toggle_icon_size - TimeoutError: wait_until timed out in 5000 milliseconds
FAILED panel/tests/ui/widgets/test_icon.py::test_toggle_icon_width_height - TimeoutError: wait_until timed out in 5000 milliseconds
FAILED panel/tests/ui/widgets/test_tabulator.py::test_tabulator_patch_no_height_resize - TimeoutError: wait_until timed out in 5000 milliseconds
FAILED panel/tests/ui/widgets/test_tabulator.py::test_tabulator_edit_event_and_header_filters - playwright._impl._errors.TimeoutError: Locator.fill: Timeout 20000ms exceeded.
================================= 5 failed, 481 passed, 11 skipped, 33 xfailed, 81 warnings, 18 rerun in 191.78s (0:03:11) ==================================
```

**Comment by hoxbro** [comment]
The error you is a bug and should be updated to `wait_until(lambda: bool(clicks) and clicks[0].option == 'B') `

Does the timeout error happen for the same tests when you rerun the code? Or is it a different test timing out?

Are you running on a slow connection internet connection or VPN?

**Comment by cdeil** [comment]
Fixed in #6909
